.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+###### Description
+
+Fill in your description here.
```

Comment:
Should we have a section to ask for some key things? Like:

How was this deployed? Helm or YAML
What version of Kubernetes are you running?
Is this a managed service, if so which one?
Please share any customizations you have made
What version of the Prometheus Operator and Fluent-Bit charts (helm ls)

Reply:
that sounds more like an issue/bug report.  This is the PR template when adding new functionality or fixing issues.

Reply:
facepalm, you are right apologies, saw PR template yet my mind went to issue template (which would be a good add to save some back and forth)